src/back/controllers/api/routes/profile/myProfile.controller.spec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const MyProfileController = require('./myProfile.controller');
+const ProfileController = require('./profile.controller');
+const profileProperties = require('./profileProperties');
+
+describe('MyProfileController', () => {
+    let sut;
+
+    beforeEach(() => {
+        sut = new MyProfileController();
+    });
+
+    it('should extend profile controller', () => {
+        sut.should.be.instanceOf(ProfileController);
```

Comment:
There are a lot of privates in ProfileController is not reused as all methods accept request and response, so the optimal decision was to use inheritance to override getById, as user in ProfileController is resolved depends on passed via request param id.

I would prefer to avoid BaseController at all, and use services approach instead.